### PR TITLE
created peerj skeleton

### LINKED ISSUE #9: Create rticles skeleton for PeerJ

### PR DESCRIPTION
**Pull request by arrismo**
closes #9

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,5 +4,6 @@
 inst/doc
 README_cache
 .DS_Store
+*.log
```
Comment (rudeboybert) [inline .gitignore:7]: good job adding `.log` to .gitignore

File: vignettes/Manuscript/Manuscript.Rmd [hunk 2]
```diff
@@ -0,0 +1,209 @@
+---
+title: Template for preparing your research report submission to PeerJ using RMarkdown
+preprint: false
+author: 
+  - name: Cici Chen
+    affiliation: 1
+    email: cicichen@u.washington.edu
+  - name: Albert Y. Kim
+    affiliation: 1
+    email: albert@stat.washington.edu
```
Comment (rudeboybert) [inline vignettes/Manuscript/Manuscript.Rmd:10]:
```suggestion
    email: akim04@smith.edu
```

**Review by rudeboybert — CHANGES_REQUESTED** [review]
Looks good overall, but please update my email address as suggested.

**Comment by arrismo** [comment]
Hey @rudeboybert I have completed your review.

**Review by rudeboybert — APPROVED** [review]

**Pull request merged**